sklearn/mixture/__init__.py

```diff
@@ -7,10 +7,18 @@
 from .gmm import _validate_covars
 from .dpgmm import DPGMM, VBGMM
 
+from .gaussianmixture import GaussianMixture
+from .bayesianmixture import BayesianGaussianMixture
+from .dpgaussianmixture import DirichletProcessGaussianMixture
```

Comment:
I would prefer the files to be named "gaussian_mixture", "bayesian_mixture" and "dp_gaussian_mixture".